### PR TITLE
AHTI-125 Add contact information

### PR DESCRIPTION
**Pull request by charn**
This PR adds contact information for features.

* Contact information is made available from the GQL API.
* Address information is fetched from MyHelsinki Places API.

### CODE REVIEW

File: features/importers/myhelsinki_places/importer.py [hunk 1]
```diff
@@ -109,6 +115,27 @@ def import_feature_tags(self, feature: Feature, tags: Iterable[dict]):
...
+        """Imports contact info for the given feature."""
+
+        if not (
+            address["street_address"]
+            or address["postal_code"]
+            or address["municipality"]
+        ):
+            # Delete existing address if source doesn't provide this information
+            if hasattr(feature, "address"):
+                feature.address.delete()
```
Comment (tuomas777) [inline features/importers/myhelsinki_places/importer.py]: I guess this should be `contact_info` instead `address`?

File: features/models.py [hunk 2]
```diff
@@ -147,3 +147,34 @@ class Meta:
                 fields=["feature", "tag"], name="unique_feature_tag"
             ),
         ]
+
+
+class ContactInfo(models.Model):
+
```
Comment (tuomas777) [inline features/models.py]: This blank line doesn't seem to match style elsewhere in this project. Not sure if I can sleep properly after seeing this.

File: features/models.py [hunk 3]
```diff
@@ -147,3 +147,34 @@ class Meta:
...
+    )
+    municipality = models.CharField(_("municipality"), max_length=64, blank=True)
+    phone_number = models.CharField(
+        verbose_name=_("phone number"), max_length=32, blank=True
+    )
+    email = models.EmailField(verbose_name=_("email"), blank=True)
+
+    class Meta:
+        verbose_name = _("address")
+        verbose_name_plural = _("addresses")
```
Comment (tuomas777) [inline features/models.py]: Is this model actually supposed to be called `address`?

File: features/models.py (hunk 2 above)
Comment (igordavydsson) [reply features/models.py]: well, now I won't be able to sleep either 😢

Comment (igordavydsson) [reply features/models.py]: barely slept 😪

Comment (charn) [reply features/models.py]: Sorry for this, I'll fix it right away!

File: features/importers/myhelsinki_places/importer.py (hunk 1 above)
Comment (charn) [reply features/importers/myhelsinki_places/importer.py]: I was supposed to fix it, but apparently I got distracted while refactoring the related if clause. :sweat:

Comment (charn) [reply features/importers/myhelsinki_places/importer.py]: Nice spot.

File: features/models.py (hunk 3 above)
Comment (charn) [reply features/models.py]: It was, until I refactored things. :sob:

File: features/importers/myhelsinki_places/importer.py [hunk 4]
```diff
@@ -109,6 +115,27 @@ def import_feature_tags(self, feature: Feature, tags: Iterable[dict]):
...
+    def import_feature_contact_info(self, feature: Feature, address: dict):
+        """Imports contact info for the given feature."""
+
+        if not (
+            address["street_address"]
+            or address["postal_code"]
+            or address["municipality"]
+        ):
+            # Delete existing address if source doesn't provide this information
+            if hasattr(feature, "address"):
```
Comment (igordavydsson) [inline features/importers/myhelsinki_places/importer.py]: should this line be also changed?

Comment (charn) [reply features/importers/myhelsinki_places/importer.py]: Fixed and added a test for this.

**Comment by charn** [comment]
Tried to fix all the typos. :sweat_smile:

**Pull request merged**